File: 0001 Code Chef/0010 CodeChef Round/implementation/0010 nodejs/code_chef_round.js
const readline = require('readline');

const rl = readline.createInterface({
    input:process.stdin,
    output:process.stdout
});

rl.question('',(input)=>{
    day = parseInt(input);
    if(day == 4){
        console.log("YES");
    }else{
        console.log("NO");
    }
    rl.close();
})

File: 0001 Code Chef/0010 CodeChef Round/implementation/0005 kotlin/CodeChefRound.kt
fun  main(args:Array<String>){
    var stringInput:String? = readLine();
    var day:Int? = stringInput?.toInt();
    if(day != null){
        if(day == 4){
            println("YES");
        }else{
            println("NO");
        }
    }
}